src/templates/console.html

```diff
@@ -66,6 +71,8 @@
         };
 
         term.handlePythonResult = function(result) {
+          c.restore_stdstreams();
+          term.resume();
```

Comment:
`console.html` is not really clean with this `restore_stdstreams()` in the JS code but they will disappear in the part 2/3 of the console improvement, I promise.